### ISSUE TITLE
ERROR:Nansat:GDAL could not open file.zip, trying to read with Nansat mappers...

### ISSUE THREAD

**Issue by Pratik-Bhatt190** [nansencenter/sentinel1denoised#95]
hi, I am facing the error:
input_zip_path = "D:\S1A_IW_GRDH_1SDV_20180102T001049_20180102T001114_019968_022025_5EC4.zip"
output_dir = "D:\ARC_GIS_PARENT"
do_s1denoise(input_zip_path, output_dir)######my code

#####returns the following error#####
01:09:08|40|nansat|_get_dataset_metadata|GDAL could not open D:\S1A_IW_GRDH_1SDV_20180102T001049_20180102T001114_019968_022025_5EC4.zip, trying to read with Nansat mappers...
ERROR:Nansat:GDAL could not open D:\S1A_IW_GRDH_1SDV_20180102T001049_20180102T001114_019968_022025_5EC4.zip, trying to read with Nansat mappers...

ValueError                                Traceback (most recent call last)
<ipython-input-15-6e84a4d97113> in <module>
      2 output_dir = "D:\ARC_GIS_PARENT"
      3 # a = gdal.Open()
----> 4 do_s1denoise(input_zip_path, output_dir)

<ipython-input-5-77f9c4f70f34> in do_s1denoise(input_zip_path, output_dir)
     33 
     34     ### Sentinel1 Denoised Processing
---> 35     s1 = Sentinel1Image(input_zip_path)  # Open the original product with the sentinel1Denoised SW
     36     # run thermal noise correction in HH polarization using ESA algorithim (No HH option in sentinel1Denoise)
     37     s0_hh = s1.remove_thermal_noise('HH', algorithim='ESA')

E:\MINI_CONDA\envs\eo\lib\site-packages\s1denoise\sentinel1image.py in __init__(self, filename, mapperName, logLevel)
     27     def __init__(self, filename, mapperName='sentinel1_l1', logLevel=30):
     28         ''' Read calibration/annotation XML files and auxiliary XML file '''
---> 29         Nansat.__init__( self, filename, mapperName=mapperName, logLevel=logLevel)
     30         if ( self.filename.split('/')[-1][4:16]
     31              not in [ 'IW_GRDH_1SDH',

E:\MINI_CONDA\envs\eo\lib\site-packages\nansat\nansat.py in __init__(self, filename, mapper, log_level, **kwargs)
    157         self._init_empty(filename, log_level)
    158         # Create VRT object with mapping of variables
--> 159         self.vrt = self._get_mapper(mapper, **kwargs)
    160 
    161     def __getitem__(self, band_id):

E:\MINI_CONDA\envs\eo\lib\site-packages\nansat\nansat.py in _get_mapper(self, mappername, **kwargs)
   1162                 # create a Mapper object and get VRT dataset from it
   1163                 try:
-> 1164                     tmp_vrt = nansatMappers[iMapper](self.filename, gdal_dataset, metadata, **kwargs)
   1165                     self.logger.info('Mapper %s - success!' % iMapper)
   1166                     self.mapper = iMapper.replace('mapper_', '')

E:\MINI_CONDA\envs\eo\lib\site-packages\nansat\mappers\mapper_sentinel1_l1.py in __init__(self, filename, gdalDataset, gdalMetadata, fast, fixgcp, **kwargs)
    101 
    102         # read annotation files
--> 103         self.annotation_data = self.read_annotation(annotation_files)
    104         if not fast and fixgcp:
    105             self.correct_geolocation_data()

E:\MINI_CONDA\envs\eo\lib\site-packages\nansat\mappers\mapper_sentinel1_l1.py in read_annotation(self, annotation_files)
    433 
    434         xml = self.read_vsi(annotation_files[0])
--> 435         xml = Node.create(xml)
    436         geolocation_points = xml.node('geolocationGrid').node('geolocationGridPointList').children
    437         # collect data from XML into dictionary with lists

E:\MINI_CONDA\envs\eo\lib\site-packages\nansat\node.py in create(dom)
    305         """
    306         if isinstance(dom, str):
--> 307             if os.path.exists(dom):
    308                 # parse input file
    309                 dom = xdm.parse(dom)

E:\MINI_CONDA\envs\eo\lib\genericpath.py in exists(path)
     17     """Test whether a path exists.  Returns False for broken symbolic links"""
     18     try:
---> 19         os.stat(path)
     20     except OSError:
     21         return False

ValueError: stat: path too long for Windows
![Screenshot (77)](https://user-images.githubusercontent.com/103387711/176763842-40335f43-03e3-4b68-addd-c45f55585e32.png)
![Screenshot (78)](https://user-images.githubusercontent.com/103387711/176763850-0038e191-6d8e-4b76-9625-c4d46b52c1d0.png)
![Screenshot (79)](https://user-images.githubusercontent.com/103387711/176763859-0c85b42d-1c20-4e49-b995-8bc9bcdc64f1.png)

**Comment by windpathing** [comment]
i got same issue！Have u solved it yet?

**Comment by Pratik-Bhatt190** [comment]
No, I am stuck with it too, I think sentinel1 denoise has some problems
with windows, though I have not tried it on linux or mac.If you find any
solution give me a knock.
